sdks/python/setup.py

```diff
@@ -87,6 +106,7 @@ def get_version():
     ]),
     setup_requires=['nose>=1.0'],
     install_requires=REQUIRED_PACKAGES,
+    extras_require=EXTRA_PACKAGES,
```

Comment:
It depends. The `extras_require` allows to specify that we want to install the "cython" version of the Python SDK, but as it's optional, it won't be installed by default. 

Running "pip install -e .[cython]", it will compile the .c files and use the "cython" version. To verify programmatically that cython is installed after the setup.py is called, let's say in travis, you could do something like this in bash:
```
python -c cython
status=$?
if [ $status -ne 0 ]; then
  echo "error: cython not installed" 
fi
```